scripts/deploy.ts

```diff
@@ -1,29 +1,58 @@
-// We require the Hardhat Runtime Environment explicitly here. This is optional
-// but useful for running the script in a standalone fashion through `node <script>`.
-//
-// When running the script with `npx hardhat run <script>` you'll find the Hardhat
-// Runtime Environment's members available in the global scope.
 import { ethers } from "hardhat";
+const hre = require("hardhat");
 
 async function main() {
-  // Hardhat always runs the compile task when running scripts with its command
-  // line interface.
-  //
-  // If this script is run directly using `node` you may want to call compile
-  // manually to make sure everything is compiled
-  // await hre.run('compile');
+  const WalletSimple = await ethers.getContractFactory("WalletSimple");
+  const walletSimple = await WalletSimple.deploy();
+  await walletSimple.deployed();
+  console.log("WalletSimple deployed to:", walletSimple.address);
 
-  // We get the contract to deploy
-  const Greeter = await ethers.getContractFactory("Greeter");
-  const greeter = await Greeter.deploy("Hello, Hardhat!");
+  const WalletFactory = await ethers.getContractFactory("WalletFactory");
+  const walletFactory = await WalletFactory.deploy(walletSimple.address);
+  await walletFactory.deployed();
+  console.log("WalletFactory deployed to:", walletFactory.address);
```

Comment:
should we also try to add deployed addresses to a file?

Reply:
We should be able to get the logs from the drone build, infra said that committing files to the repo will be difficult from a security perspective.